examples/shallow_water.py

```diff
@@ -56,7 +56,7 @@
 nproc_y = min(mpi_size, 2)
 nproc_x = mpi_size // nproc_y
 
-proc_idx = jnp.unravel_index(mpi_rank, (nproc_y, nproc_x))
+proc_idx = np.unravel_index(mpi_rank, (nproc_y, nproc_x))
```

Comment:
Did this break?

Reply:
Yes. It was giving a concretization error when you compare it with an int, a bit lower down.

Reply:
(I’m surprised it was working before)

Reply:
Since this is outside of JIT I feel like this should work. Oh, well 🤷